### DISCUSSION TITLE
Roll Damping

### DISCUSSION THREAD

**Discussion by zorareh** [capytaine/capytaine#579] [General]
Hi, How to define roll damping (or percentage of critical damping), in order to calculte roll peak value correctly?

**Comment by baptistelabat-syroco** [comment]
I am facing the same question. 

You can have a look to https://github.com/pciuh/roll-damping-streamlit or https://github.com/pciuh/roll-damping to estimate the dimensionful B44 term.

Then I propose the following code snippet, but I would like somebody else to validate it !

from capytaine.post_pro import rao
roll_damping = 1e8 # N.m/(rad/s)
viscous_damping = dataset.radiation_damping*0
viscous_damping.loc[dict(influenced_dof='Roll', radiating_dof='Roll')]=roll_damping
rao = rao(dataset, dissipation=viscous_damping)
